docs/admin/federation/index.md

```diff
@@ -15,72 +16,129 @@ This guide explains how to set up cluster federation that lets us control multip
 This guide assumes that we have a running Kubernetes cluster.
 If not, then head over to the [getting started guides](/docs/getting-started-guides/) to bring up a cluster.
 
-This guide also assumes that we have the Kubernetes source code that can be
-[downloaded from here](/docs/getting-started-guides/binary_release/).
+This guide also assumes that we have a Kubernetes release
```

Comment:
Avoid "we". Address the user directly as "you".

This is general style guideline for all the docs. I know we do it everywhere right now, but we should get started on changing that.